js/addToTask.js

```diff
@@ -9,14 +9,15 @@ import {
   TODAY,
   COMPLETE,
   EDIT,
-  DELET,
+  DELETE,
   CLICK_EVENT,
   HIDE,
   AFTER_BEGIN,
 } from "/js/constants.js"; //named import
+import removeTodos from "/js/deleteTask.js";
 
-export let taskList = [];
-export let id = 0;
+let taskList = [];
```

Comment:
if we are not reassigning the variable i.e `taskList = [...]`, consider using `const` binding over `let`

Reply:
we are updating taskList array while removing tasks as well as updating properties while performing make task state to done, edit any Task.